### DISCUSSION TITLE
Want to get mouse focus into a newly created pane

### DISCUSSION THREAD

**Discussion by manfredlotz** [wezterm/wezterm#6487] [Q&A]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

kwin_wayland

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I have defined a key binding to create a horizontal split like this

```
...
		{ key = "F6", action = wezterm.action.SplitVertical({ domain = "CurrentPaneDomain" }) },
...
```
which works fine.

But how can I get the mouse focus automatically into the newly created right pane?

**Comment by bew** [comment]
I'm sorry but I don't understand what you want here 🤔
What exactly is the issue with the mouse?

**Comment by manfredlotz** [comment]
I think what I want is simply not possible to do.

When the mouse focus is in the wezterm terminal I can select text. When I do a SplitVertical my keyboard focus switches to the new pane which is what I want. Additionally, I was thinking if it is possible to get the mouse focus moved to the new pane as well. Now, if for some reason I like to select text in the new pane I first have to move the mouse pointer to the new pane.

**Reply by bew** [reply]
Ah so move the mouse pointer to the focused pane, yeah that's not possible with wezterm currently.
